spec/models/task_spec.rb

```diff
@@ -8,6 +8,10 @@
 
   it { is_expected.to have_many(:submissions) }
 
+  it "defaults to an 'unstarted' state" do
+    expect(Task.new.status).to eq "unstarted"
```

Comment:
`unstarted` should be single-quoted - as it's currently the Rubocop style check on CI.